zproject/urls.py

```diff
@@ -226,7 +226,7 @@
          'DELETE': 'zerver.views.users.deactivate_user_own_backend'}),
     url(r'^users/me/pointer$', rest_dispatch,
         {'GET': 'zerver.views.pointer.get_pointer_backend',
-         'PUT': 'zerver.views.pointer.update_pointer_backend'}),
+         'POST': 'zerver.views.pointer.update_pointer_backend'}),
```

Comment:
The rest are fine to change, but we can't remove the old `update_pointer_backend`, since it's used by the mobile apps.  I think we should probably support both PUT and POST for this, and add a comment here explaining that users should use the POST version, and we intend to remove the PUT version once we can.